src/commands/createFunction/FunctionListStep.ts

```diff
@@ -164,6 +176,14 @@ export class FunctionListStep extends AzureWizardPromptStep<IFunctionWizardConte
             });
         }
 
+        if (language === ProjectLanguage.CSharp || language === ProjectLanguage.Java || language === ProjectLanguage.Python || language === ProjectLanguage.TypeScript) {
```

Comment:
FunctionListStep is used for 'Create Project' _and_ 'Create Function'. Seems like you were saying this is only for 'Create Project', in which case you should check `this._isProjectWizard` in this conditional

Reply:
I started investigating after looking at your comment. I feel this is useful in both the cases - and your hunch was correct. As people can _Skip for now_, the access to creating HttpTriggers from OpenAPI goes away, and that is not the ideal. Which is why I moved this out of the `this._isProjectWizard` conditional.